plugins/woocommerce/src/Admin/API/Reports/Categories/DataStore.php

```diff
@@ -201,99 +199,88 @@ protected function include_extended_info( &$categories_data, $query_args ) {
 	}
 
 	/**
-	 * Returns the report data based on parameters supplied by the user.
+	 * Get the default query arguments to be used by get_data().
+	 * These defaults are only partially applied when used via REST API, as that has its own defaults.
 	 *
-	 * @param array $query_args  Query parameters.
-	 * @return stdClass|WP_Error Data.
+	 * @return array Query parameters.
+	 */
+	public function get_default_query_vars() {
+		$defaults                      = parent::get_default_query_vars();
+		$defaults['category_includes'] = array();
+		$defaults['extended_info']     = false;
+
+		return $defaults;
+	}
```

Comment:
:scroll: I wonder if we really need to handle default query vars here and in the Query class. Maybe we could simplify it to just one source.